src/main/java/home/intexsoft/bank_application/command/AddMoneyTransferCommand.java

```diff
@@ -0,0 +1,73 @@
+package home.intexsoft.bank_application.command;
+
+import home.intexsoft.bank_application.dto.ActionDto;
+import home.intexsoft.bank_application.dto.OperationDto;
+import home.intexsoft.bank_application.service.OperationService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AddMoneyTransferCommand extends Command {
+
+    private static final Logger log = LoggerFactory.getLogger(AddMoneyTransferCommand.class);
+
+    public enum Attribute implements CommandAttribute {
+
+        SENDER_BANK_ACCOUNT_ID("sender bank account id"),
+        RECIPIENT_BANK_ACCOUNT_ID("recipient bank account id"),
+        AMOUNT_OF_MONEY("amount of money");
+
+        private String attributeName;
+
+        Attribute(String attributeName) {
+            this.attributeName = attributeName;
+        }
+
+        public String getAttributeName() {
+            return attributeName;
+        }
+    }
+
+    {
```

Comment:
@DmitryZagorski, move all initialization blocks to the top, right after class declaration.